releases/keycloak-gatekeeper.yaml

```diff
@@ -0,0 +1,113 @@
+repositories:
+# No official chart, this user's chart seems to be the best there is
+- name: "gabibbo97"
+  url: "https://gabibbo97.github.io/charts/"
+
+######
+# You need to create a gatekeeper.yaml file with an array of services, with values that do not use go templating
+# Here is an example:, with comments. Most elements can be omitted
+# services:
+#    - name: dashboard  # the service name
+#      portalName: "Kubernetes Dashboard"  # The name as it should appear in the Forecastle portal (omit to hide)
+#      host: dashboard.us-west-2.corp.cpco.io # The DNS name of the public endpoint
+#      useTLS: false  # True to generate a TLS certificate and use an HTTPS endpoint
+#      upstream: https://kubernetes-dashboard.kube-system.svc.cluster.local  # The service URL
+#      skipUpstreamTlsVerify: true # True to skip the validity check of the TLS certificate of the upstream URL
+#      rules: # Array of access control rules
+#        - "uri=/*|roles=kube-admin"
+#      replicas: 1 # number of replicas
+#    - name: forecastle
+#      debug: true
+#      host: forecastle.us-west-2.corp.cpco.io
+#      upstream: http://forecastle.kube-system.svc.cluster.local
+#      rules:
+#        - "uri=/*|roles=kube-admin,user|require-any-role=true"
+######
+
+#
+environments:
+  default:
+    values:
```

Comment:
We've been sticking values here: 

https://github.com/cloudposse/helmfiles/tree/master/releases/values

Reply:
These values go in Geodesic, they are not part of the release

Reply:
right, but the user should stick them in `/conf/helmfiles/values`

Reply:
if you look in `values/` those are all *examples*

Reply:
Well, at this point `values/` is mostly templates that are used by the helmfiles, but I can move this if you want.